### PR TITLE
Add cpulimit SPEC to Mariner

### PR DESCRIPTION
**Pull request by chalamalasetty**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 
Add cpulimit SPEC to Mariner.

###### Change Log  

- Change
  Add cpulimit SPEC to Mariner

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: Local build.

### CODE REVIEW

File: SPECS/cpulimit/cpulimit.spec [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+Name:           cpulimit
+Version:        2.8
+Release:        2%{?dist}
+Summary:        Limit the CPU Usage of a Process
+License:        GPL-2.0-or-later
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System/Monitoring
+URL:            http://limitcpu.sourceforge.net/
```
Comment (mfrw) [inline SPECS/cpulimit/cpulimit.spec:28]:
```suggestion
URL:            https://limitcpu.sourceforge.net/
```

File: SPECS/cpulimit/cpulimit.spec [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+Source0:        https://downloads.sourceforge.net/limitcpu/%{name}-%{version}.tar.gz
+Patch0:         %{name}-2.2-do_not_forget_version.patch
```
Comment (mfrw) [inline SPECS/cpulimit/cpulimit.spec:30]: NIT: Given the patch does `2.8` and not `2.2`
```suggestion
Patch0:         %{name}-2.8-do_not_forget_version.patch
```

File: SPECS/cpulimit/cpulimit.spec [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+has plenty of CPU cycles available for other tasks. It can also be used
+to keep laptops cool in the face of CPU-hungry processes and for limiting
+virtual machines.
+
+LimitCPU is the direct child of CPUlimit, a creation of Angelo Marletta,
+which can be found at http://cpulimit.sourceforge.net
+
+%prep
+%setup -q
+%patch0 -p1
```
Comment (mfrw) [inline SPECS/cpulimit/cpulimit.spec:48]: Can we please switch to `autosetup`
Makes further CVE maintaince simpler

File: SPECS/cpulimit/cpulimit.spec [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+Name:           cpulimit
+Version:        2.8
+Release:        2%{?dist}
+Summary:        Limit the CPU Usage of a Process
```
Comment (mfrw) [inline SPECS/cpulimit/cpulimit.spec:23]: I think we should honor the `linter` and make `Summary` as the first line :)

**Review by anphel31 — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks

**Pull request merged**